core/domain/story_jobs_one_off_test.py

```diff
@@ -162,8 +162,27 @@ def test_migration_job_converts_old_story(self):
         story = (
             story_models.StoryModel.get(self.STORY_ID))
         self.assertEqual(story.story_contents_schema_version, 1)
+        self.assertEqual(
+            story.story_contents,
```

Comment:
Seems off. Why is this story.story_contents and below it's story.story_contents.to_dict()?

(Make sure you always **locally** run the backend tests corresponding to the code you write, before you push.)

Reply:
The story.story_contents in L165 didn't require a to_dict() because it was not a model.

(I did test this locally and that is how I figured this didn't require a to_dict() to work.)

Reply:
But then why isn't story.story_contents a dict in line 184? That seems wrong -- it shouldn't change type. /cc @aks681 

EDIT: Ah, I know what's going wrong here. The naming is misleading! Please name line 162 story_model instead. It's very important to differentiate clearly between the model instance and the domain object.